.github/workflows/standard_spa_pr.yml

```diff
@@ -228,14 +255,12 @@ jobs:
     concurrency:
       group: deploy-magic-url-${{ needs.create-shared-vars.outputs.groupId }}
       cancel-in-progress: true
-    needs: [create-shared-vars]
+    needs: [create-shared-vars, migration_number]
     if: ${{ inputs.use_magic_url }}
     permissions:
       id-token: write
       contents: read
       pull-requests: write
-    outputs:
```

Comment:
This logic got moved into its own job (migration_number) as we now have multiple jobs relying on it